client/state/data-layer/wpcom/sites/activity/index.js

```diff
@@ -83,6 +83,9 @@ export const continuePolling = ( { dispatch }, action ) => {
 		}
 
 		const timer = setTimeout( () => {
+			if ( ! pollingSites.get( siteId ) ) {
```

Comment:
It doesn't seem obvious to me why this is here. Think it would be good to leave in the comment explaining what's going on. Your call